### PR TITLE
Fix level blueprints for both schema generation and object look up.

### PR DESCRIPTION
**Pull request by Vatyx**
#### Description
A level blueprint class has trouble being loaded with a `LoadObject` call (it crashes the editor). So during schema generation we skip over them in the SchemaDatabase. This meant we could only support level blueprints for one map.

We now don't delete generated schema, so if a previous level blueprint existed, it can still be used after generating schema for a different map.

Another issue was that the class that was associated with a level blueprint could be PIE prefixed in cases where there were sublevels, but not PIE prefixed when there was only a persistent level. Due to this, if an Actor is stably named, we look up the Actor rather than trying to load the class and then look up the Actor. This means for level blueprints, they will always be looked up first and for other Actors, we will load their class.

#### Release note
Bugfix: Support level blueprints for multiple levels.

#### Tests
Tested locally between two maps. Will test on ShooterGame to make sure stuff still works.

#### Primary reviewers
@m-samiec @BenNaccarato

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h [hunk 1]
```diff
@@ -144,8 +144,9 @@ class USpatialReceiver : public UObject
 	void RemoveActor(Worker_EntityId EntityId);
 	void DestroyActor(AActor* Actor, Worker_EntityId EntityId);
 
-	AActor* GetOrCreateActor(improbable::UnrealMetadata* UnrealMetadata, UClass* ActorClass, improbable::SpawnData* SpawnData);
-	AActor* CreateActor(improbable::SpawnData* SpawnData, UClass* ActorClass);
+	AActor* TryGetOrCreateActor(improbable::UnrealMetadata* UnrealMetadata, improbable::SpawnData* SpawnData);
+	AActor* CreateActor(improbable::UnrealMetadata* UnrealMetadata, improbable::SpawnData* SpawnData);
+	AActor* SpawnActor(improbable::SpawnData* SpawnData, UClass* Class);
```
Comment (m-samiec) [inline SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h]: nit: I don't think the SpawnActor function is necessary. It's functionality bleeds into CreateActor and creates a confusing interface.

Comment (Vatyx) [reply SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h]: hmmmmmmMMMMMMMMMMMMM fair point.

File: SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealMetadata.h [hunk 2]
```diff
@@ -38,6 +38,11 @@ struct UnrealMetadata : Component
 		}
 		OwnerWorkerAttribute = GetStringFromSchema(ComponentObject, 2);
 		ClassPath = GetStringFromSchema(ComponentObject, 3);
+
+		if (Schema_GetBoolCount(ComponentObject, 4) == 1)
```
Comment (BenNaccarato) [inline SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealMetadata.h]: Is it worth making spatial constants for these, to avoid magic numbers?

Comment (Vatyx) [reply SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealMetadata.h]: Yes. That should be done in a different task I think.

**Pull request merged**